libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/map/NavigationMapboxMap.java

```diff
@@ -51,11 +49,11 @@
 public class NavigationMapboxMap {
 
   static final String STREETS_LAYER_ID = "streetsLayer";
-  private static final String MAPBOX_STREETS_V7 = "mapbox://mapbox.mapbox-streets-v7";
+  private static final String MAPBOX_STREETS_V7_URL = "mapbox://mapbox.mapbox-streets-v7";
+  private static final String MAPBOX_STREETS_V8_URL = "mapbox://mapbox.mapbox-streets-v8";
   private static final String STREETS_SOURCE_ID = "streetsSource";
```

Comment:
Might want to take this opportunity to make the source ID more unique, like `com.mapbox.navigation.streets`.